dpipe/dpipe.go

```diff
@@ -0,0 +1,144 @@
+// Package dpipe provides the pipe works like datagram protocol on memory.
```

Comment:
Please add a note to indicate this package is mainly for testing and not for production

Reply:
Thanks :) Done!